packages/webapp/src/containers/Profile/People/saga.js

```diff
@@ -125,6 +125,7 @@ export function* invitePseudoUserSaga({ payload: user }) {
       user,
       header,
     );
+    localStorage.setItem('litefarm_lang', user.language);
```

Comment:
@seinnlee Can you explain why we set the user's language here?

Reply:
You're right. This shouldn't be here. This only work for local environment.

I made a new change and pushed it. Can you review again? (I'm using url param from email invite link to change the app language)